### ISSUE TITLE
Create an empty repository by executing "hub create"

### ISSUE THREAD

**Issue by shigemk2** [github/hub#1162]
Normally, we use "hub create" to create new repository.

For example:
```sh
$ mkdir test1
$ cd test1/
$ git init && git add . && git commit --allow-empty -m "Initial commit"
$ hub create 
Updating origin
created repository: XXXXX/test1
```

Current directory have to be a git repo now. But I want to create an empty repository with "hub create".

Just like this:

```sh
$ hub create -n shigemk2/test1
Updating origin
created repository: XXXXX/test1
```

Current directory is able to be anywhere(It doesn't have to be a git repo!).

Is this feature in demand?

**Comment by mislav** [comment]
That's a good feature idea. I'll think about it.

Somewhat related: https://github.com/github/hub/issues/958

**Comment by boncey** [comment]
I'd like this too.

Now that GitHub is allowing [unlimited private repositories](https://github.com/blog/2164-introducing-unlimited-private-repositories) I'm moving a bunch of self-hosted repos over.
I'm working on a script to do this and being able to create repos without side-effects would be useful.

**Comment by isketerol64** [comment]
Yes, please I would love to be able to create repositories from the command line.  As it stands, I have to be in a git repo to do this with "hub" and then it ends up setting the origin to that repo, which is not what is intended at all.

"hub" is a fantastic extension to git, this feature would make it all the better again.

**Comment by fandy** [comment]
Has this feature been implemented yet? In the docs it mentions:

> Maintaining a project is easier when you can easily fetch from other forks, review pull requests and cherry-pick URLs. You can even create a new repo for your next thing.

But I couldn't find how to use the "create new repo" feature.

**Comment by mislav** [comment]
@fandy The documentation refers to the old way of creating repos on the command line, which requires having an already initialized git repository (`hub create` only creates the repository on GitHub, not locally). These would be the steps:

```sh
mkdir myproject
cd myproject
git init
# ...create some files, commits...

hub create
git push -u origin HEAD
```

However, this thread is about proposed new functionality to hub (not yet implemented) which would **both** create the repository on GitHub and locally in a new directory.

**Comment by fandy** [comment]
Thanks for clarifying! That does seem like a useful feature.

**Comment by igbanam** [comment]
This sounds like a good idea. But this may blur the lines between git and hub.

With this request, we are trying to make hub do git stuff. I had originally thought that hub was focused on GitHub only.

To achieve this, you could create an orchestrator (maybe a simple shell script) which makes use of both git and hub to achieve this?
